spec/services/et_atos_export/export_service_spec.rb

```diff
@@ -2,16 +2,18 @@
 
```

Comment:
The export service spec now proves that the exported_file is marked with the correct external_system_id